### ISSUE TITLE
Better error messages when a external dependency isn't found

### ISSUE THREAD

**Issue by dstufft** [pypa/packaging-problems#27]
Is there any way to give a better error message when an external dependency, like libxml2, is not found for a project?

**Comment by brainwane** [comment]
Is this still an issue?

**Comment by ncoghlan** [comment]
Yep. When installing from source, you'll get a cryptic compile error, while installing from a wheel file will give an import error when the relevant extension module fails to load.

**Comment by njsmith** [comment]
It has become less common, though, now that many projects provide self-contained wheels.

**Comment by pradyunsg** [comment]
Is this still relevant?

**Comment by merwok** [comment]
The group at https://github.com/pypackaging-native/ is trying to wrangle this area of issues.

**Comment by merwok** [comment]
[PEP 725 – Specifying external dependencies in pyproject.toml](https://peps.python.org/pep-0725/) is trying to address this! Closing this ticket in favour of the PEP and its discussions.
